pulpcore/tasking/entrypoint.py

```
@@ -13,12 +12,10 @@
 
 django.setup()
 
-from django.conf import settings  # noqa: E402: module level not at top of file
-
-
 _logger = logging.getLogger(__name__)
 
 
+# TODO Are we ready to remove this option?
```

I think so, yes.

---

It will break systemd files for the resource-manager that are deployed in all different installer scenarios.
So the question to me is how much do we want to break those to force them fixing it? Or should we warn one release longer, now that you cannot even start a resource-mananger if you wanted?

---

Personally I'm in favor of removing it. It's going to be a breaking change at some point anyway. It shouldn't have a very large impact since most users will have this auto-handled for them by the installer (which would need a corresponding update in the same release for sure). That's my opinion anyway. If others really want to leave it I'm ok with that too.

---

And that is exactly my point. We would have less churn if we give the installer one release cycle to remove that service.

The reason, why i am cautious is that i think if i remove the option and the user attempts to start a resource-manager, it will fail hard ad respawn the service in a busy loop instead of just starting an additional worker (In the end both are undesirable, but the latter is a little less so.).

---

I imagined the installer would have to handle this in the same release. I'm ok with either approach. Can you decide and file/update the redmine tasks with the corresponding releases?

---

Thanks @mdellweg . I updated that issue with the 3.17.0 milestone so that it will formally block that release.